### ISSUE TITLE
Multiple instances of client

### ISSUE THREAD

**Issue by captainjackrana** [lbdremy/solr-node-client#190]
I have a user facing application where people make searches on a particular collection. 
How is the multi-request connection handled. Should i be creating new client instances on every request?

**Comment by ColadaFF** [comment]
Not at all, remember that this client uses HTTP for comunication, so, one instance per core should be fine.
The thing I would be careful of is that solr has a number of searchers by default, so if a lot of clients are searching at the same time, you could get an exception response from solr.

You can read more about it here: [SolrDocs](https://cwiki.apache.org/confluence/display/solr/Query+Settings+in+SolrConfig)
